src/screens/Dashboard/index.js

```diff
@@ -1,30 +1,38 @@
 import React from 'react'
+import {hasUnlockedPublished} from 'utils/milestones'
 import LayoutColumns from 'components/LayoutColumns'
 import InstructorStatsCard from 'components/InstructorStatsCard'
 import InstructorLessonsCard from 'components/InstructorLessonsCard'
 import RequestedLessonsCard from 'components/RequestedLessonsCard'
-import ProposeLessonCard from 'components/ProposeLessonCard'
 import GetPublishedCard from './components/GetPublishedCard'
 import HelpCard from './components/HelpCard'
 import InstructorRevenueCard from './components/InstructorRevenueCard'
 
-export default ({instructor}) => (
-  <div>
-    <LayoutColumns items={[
-      <GetPublishedCard instructor={instructor} />,
-      <HelpCard publishedLessons={instructor.published_lessons} />,
-    ]} />
-    <LayoutColumns items={[
-      <InstructorStatsCard
-        publishedLessons={instructor.published_lessons}
-        publishedCourses={instructor.published_courses}
-      />,
-      <InstructorRevenueCard revenueUrl={instructor.revenue_url} />,
-    ]} />
-    <InstructorLessonsCard instructor={instructor} />
-    <LayoutColumns items={[
-      <RequestedLessonsCard instructor={instructor} />,
-      <ProposeLessonCard instructor={instructor} />,
-    ]} />
-  </div>
-)
+export default ({instructor}) => hasUnlockedPublished(instructor.published_lessons)
+  ? <div>
+      <LayoutColumns 
+        items={[
+          <InstructorRevenueCard revenueUrl={instructor.revenue_url} />,
+          <InstructorStatsCard
+            publishedLessons={instructor.published_lessons}
+            publishedCourses={instructor.published_courses}
+          />,
+        ]}
+        relativeSizes={[2, 1]}
+      />
+      <LayoutColumns 
+        items={[
+          <InstructorLessonsCard instructor={instructor} />,
+          <RequestedLessonsCard instructor={instructor} />,
+        ]}
+        relativeSizes={[2, 1]}
+      />
+    </div>
+  : <div>
+      <LayoutColumns items={[
+        <GetPublishedCard instructor={instructor} />,
+        <RequestedLessonsCard instructor={instructor} />,
+        <HelpCard publishedLessons={instructor.published_lessons} />,
+      ]} />
+      <InstructorLessonsCard instructor={instructor} />
+    </div>
```

Comment:
Changes dashboard layout depending on if instructor is published or not:

Unpublished:
![image](https://cloud.githubusercontent.com/assets/5497885/24430174/97667d88-13d2-11e7-9502-95d29b3e57c0.png)

Published:
![image](https://cloud.githubusercontent.com/assets/5497885/24430192/a82ee600-13d2-11e7-845d-a76d4bfe2088.png)

This matches the latest mock layouts; the individual components inside will be updated in separate PRs.